package.json

```diff
@@ -84,7 +84,7 @@
     "comma-separated-tokens": "^2.0.0",
     "prop-types": "^15.0.0",
     "property-information": "^6.0.0",
-    "react-is": "^17.0.0",
+    "react-is": ">=16",
```

Comment:
:thinking: If we just unlock the version, npm and yarn will still install latest by default, wouldn't they?
I wonder if this should be moved to a peerDependency?